### DISCUSSION TITLE
Option to only format specific tags

### DISCUSSION THREAD

**Discussion by Shrugsy** [hosseinmd/prettier-plugin-jsdoc#106] [Ideas]
Is it possible to provide an option to only format specific tags, rather than the whole docblock?  

I would like to take advantage of the formatting of content under the `@example` tag, while leaving the rest of the docstring untouched.

e.g. for the input below:
```ts
/**
 * This is a summary for foo.
 * 
 * foo is the name of the function.
 * 
 * @remarks
 * This is some additional info
 * 
 * 1. point 1
 * 2. point 2
 * 3. point 3
 * 
 * @example
 * ```ts
 * foo(2, 5,    9)
 * ```
 */
function foo(num1, num2, num3) {
 //
}
```

I would like only the @example tag formatted. Currently, it formats the summary & remarks as well like so:
```
/**
 * This is a summary for foo.
 *
 * Foo is the name of the function.
 *
 * @remarks This is some additional info
 *
 *   1. Point 1
 *   2. Point 2
 *   3. Point 3
 * @example
 *
 * ```ts
 * foo(2, 5, 9);
 * ```
 */
function foo(num1, num2, num3) {
  //
}
```

**Comment by fardad-dev** [comment]
I think we could add an option to add a list as pre formatted tags

But i would like to know what part of formatted code bothers you?

**Reply by Shrugsy** [reply]
For the example above:
- Text moves inline with the `@remarks` tag
- Without the text 'This is some additional info' for the `@remarks` tag, the three 'points' get moved onto a single line inline with the `@remarks` tag
- The first letter of the sentence becomes capitalized (e.g. `foo` becomes `Foo`, even though it is referring to the lowercase function `foo`
- For this example, after formatting twice, it capitalizes the second word for the remark, so that `This is` becomes `This Is`

And not shown in the example, but double-space for newlines gets consumed upon formatting, moving multiple things onto one line.

The main thing is that I was specifically looking for a tool to just format `@example` snippets, which this does well, but I had no intent to format the rest of the docblock.

**Reply by fardad-dev** [reply]
I fixed them in this commit https://github.com/hosseinmd/prettier-plugin-jsdoc/commit/4e46c4f3b4a9ad4aa3ff2bd0a4f89790457d5f11.
I will publish soon
